packages/web/src/scss/components/Tooltip/README.md

```diff
@@ -224,14 +224,49 @@ on the `.Tooltip__arrow` to control it and prevent conflicts with the default CS
 </div>
 ```
 
-### Example
+### Floating UI functionality
```

Comment:
```suggestion
### Advanced floating functionality
```

I think nobody cares for underlying implementation.

Reply:
changed, thank you